### PR TITLE
RHEL 8 & iOS client instructions for #164

### PR DESCRIPTION
**Pull request by dvershinin**
After quite some frustration, it works for me. Compiled instructions for RHEL 8 & iOS for the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,27 @@ It will install WireGuard (kernel module and tools) on the server, configure it,
...
+
+```
+sudo dnf install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm https://www.elrepo.org/elrepo-release-8.el8.elrepo.noarch.rpm
+sudo dnf install dnf-plugins-core
+sudo dnf config-manager --enable elrepo
+sudo dnf install kmod-wireguard wireguard-tools
+```
+
+Create file `/etc/wireguard/wg0.conf` with contents of the client config generated by the script (copy paste from server).
+
```
Comment (linux-modder) [inline README.md:53]: Can also just cp /mv wg0-client-foo.conf to /etc/wireguard/wg0.conf as well.  I recommend cp as you can use the original as a safety or template to hack on for tweaks/troubleshooting.

File: README.md [hunk 2]
```diff
@@ -36,6 +36,32 @@ It will install WireGuard (kernel module and tools) on the server, configure it,
...
+sudo dnf install dnf-plugins-core
+sudo dnf config-manager --enable elrepo
+sudo dnf install kmod-wireguard wireguard-tools
+```
+
+* Create file `/etc/wireguard/wg0.conf` with contents of the client config generated by the script (copy paste from server).
+
+* Connect: `systemctl start wg-quick@wg0.service`
+
+* Verify your public IP has changed to the server's IP, e.g.: `curl https://ident.me`
```
Comment (jaminmc) [inline README.md:56]:
```suggestion
* Verify your public IP has changed to the server's IP, e.g.: `curl ipv4.icanhazip.com ; curl ipv6.icanhazip.com`
```
Ident.me is blocked by pihole, so if a pihole dns is used, the command won't work. It does work with the default DNS servers. This solution will show both ipv4 and ipv6 addresses.

**Review by jaminmc — COMMENTED** [review]
Just a couple changes :)

File: README.md [hunk 3]
```diff
@@ -36,6 +36,32 @@ It will install WireGuard (kernel module and tools) on the server, configure it,
...
+* Install the necessary packages:
+
+```
+sudo dnf install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm https://www.elrepo.org/elrepo-release-8.el8.elrepo.noarch.rpm
+sudo dnf install dnf-plugins-core
+sudo dnf config-manager --enable elrepo
+sudo dnf install kmod-wireguard wireguard-tools
+```
+
+* Create file `/etc/wireguard/wg0.conf` with contents of the client config generated by the script (copy paste from server).
```
Comment (jaminmc) [inline README.md:52]:
```suggestion
* Create file in `/etc/wireguard/` with the name you want the network interface to be with extension of `.conf`. IE: `/etc/wireguard/wg0.conf`, then put in the contents of the client config generated by the script (copy paste from server). Also run `sudo chown -R root. /etc/wireguard ; chmod -R 600 /etc/wireguard` for proper permissions.

To enable the connection run `sudo wg-quick up [Interface name]`, like `sudo wg-quick up wg0` and to disable `wg-quick down [Interface name]`
```
This allows for multiple WireGuard VPN's and user customization. And how they can enable and disable

**Pull request closed without merging**